src/app/payment-hub/request-to-pay/incoming-request-to-pay/incoming-request-to-pay.component.html

```diff
@@ -15,8 +15,12 @@
   </mat-form-field>
 
   <mat-form-field fxFlex="20%">
-    <mat-label>Payer DFSP Id</mat-label>
-    <input matInput [formControl]="payerDfspId">
+    <mat-label>AMS Business Short Code</mat-label>
+    <mat-select [formControl]="payerDfspId">
+      <mat-option *ngFor="let code of amsCodes" [value]="code.value">
```

Comment:
Looks like everything in `amsCodes`  will be available in the options here. I think only TILL numbers should be shown for `incoming request to pay `